.gitignore

```diff
@@ -1,25 +1,36 @@
+build/
+bin/
+lib/
```

Comment:
I would remove `build`, as this is definitely not required. However, I'd keep `bin` and `lib`. This does not hurt and actually forces us to think about if what we are doing makes sense.

As soon as you have added something in those folders (with `git add -f`),  those files are tracked regularly.